app/lib/irs1040_schedule_eic_pdf.rb

```diff
@@ -30,10 +30,12 @@ def dependents_info(dependent_nodes)
         answers["ChildBirthYr#{index + 1}[1]"] = dependent.at("ChildBirthYr").text[1]
         answers["ChildBirthYr#{index + 1}[2]"] = dependent.at("ChildBirthYr").text[2]
         answers["ChildBirthYr#{index + 1}[3]"] = dependent.at("ChildBirthYr").text[3]
-        answers["ChildIsAStudentUnder24IndYes#{index + 1}"] =  xml_check_to_bool(dependent.at("ChildIsAStudentUnder24Ind")) ? "Yes" : nil
-        answers["ChildIsAStudentUnder24IndNo#{index + 1}"] = !xml_check_to_bool(dependent.at("ChildIsAStudentUnder24Ind")) ? "Yes" : nil
-        answers["ChildPermanentlyDisabledIndYes#{index + 1}"] = xml_check_to_bool(dependent.at("ChildPermanentlyDisabledInd")) ? "Yes" : nil
-        answers["ChildPermanentlyDisabledIndNo#{index + 1}"] = !xml_check_to_bool(dependent.at("ChildPermanentlyDisabledInd")) ? "Yes" : nil
+        bool = xml_bool_to_bool(dependent.at("ChildIsAStudentUnder24Ind"))
```

Comment:
@jenny-heath @mquadri23 I had to change these lines since it's a `BooleanType` not `CheckboxType` in the XSD.